source/windows/secure_channel_tls_handler.c

```diff
@@ -94,7 +94,10 @@ struct secure_channel_handler {
     struct aws_byte_buf buffered_read_in_data_buf;
     size_t estimated_incomplete_size;
     size_t read_extra;
-    uint8_t buffered_read_out_data[READ_OUT_SIZE];
+    /* This is to accomodate the extra head room we added above.
+       because we're allowing for splts, we may have more data decrypted

```

Comment:
typo splits?

Reply:
Ha, I totally missed that, my brain fixed it :(